src/sentry/static/sentry/app/views/onboarding/projectSetup.tsx

```diff
@@ -139,56 +139,96 @@ class ProjectDocs extends React.Component<Props, State> {
   }
 
   render() {
-    const {organization, project, platform, scrollTargetId} = this.props;
+    const {organization, project, platform} = this.props;
     const {loadedPlatform, platformDocs, hasError} = this.state;
 
+    const currentPlatform = loadedPlatform ?? platform ?? 'other';
+
     const introduction = (
-      <Panel>
-        <PanelBody withPadding>
-          <PlatformHeading platform={loadedPlatform ?? platform ?? 'other'} />
-
-          <Description id={scrollTargetId}>
-            {tct(
-              `Follow these instructions to install and verify the integration
-               of Sentry into your application, including sending
-               [strong:your first event] from your development environment. See
-               the full documentation for additional configuration, platform
-               features, and methods of sending events.`,
-              {strong: <strong />}
-            )}
-          </Description>
-          <Footer>
-            {project && (
-              <FirstEventIndicator
-                organization={organization}
-                project={project}
-                eventType="error"
-              />
+      <React.Fragment>
+        <TitleContainer>
+          <StepHeading step={2}>
+            {t(
+              'Prepare the %s SDK',
+              platforms.find(p => p.id === currentPlatform)?.name ?? ''
             )}
-            <div>
-              <Button
-                external
-                onClick={this.handleFullDocsClick}
-                href={platformDocs?.link}
-                size="small"
-              >
-                {t('Full Documentation')}
-              </Button>
-            </div>
-          </Footer>
-        </PanelBody>
-      </Panel>
+          </StepHeading>
+          <motion.div
+            variants={{
+              initial: {opacity: 0, x: 20},
+              animate: {opacity: 1, x: 0},
+              exit: {opacity: 0},
+            }}
+          >
+            <PlatformIcon size={48} format="lg" platform={currentPlatform} />
+          </motion.div>
+        </TitleContainer>
+        <motion.p
+          variants={{
+            initial: {opacity: 0},
+            animate: {opacity: 1},
+            exit: {opacity: 0},
+          }}
+        >
+          {tct(
+            'Don’t have a relationship with your terminal? [link:Invite your team instead].',
```

Comment:
Should this be an ascii quote?
```suggestion
            "Don't have a relationship with your terminal? [link:Invite your team instead].",
```